README.md

```diff
@@ -351,7 +351,8 @@ var result = sass.renderSync({
   data: 'body{background:blue; a{color:black;}}',
   outputStyle: 'compressed',
   outFile: '/to/my/output.css',
-  sourceMap: true, // or an absolute or relative (to outFile) path
+  sourceMap: true, // or an absolute or relative (to outFile) path,
+  sourceComments: true,  
```

Comment:
Not needed since the Sass project doesn't recommend using this option anymore (AFIK)